.github/release-drafter.yml

```diff
@@ -0,0 +1,27 @@
+version-template: '$MAJOR.$MINOR-alpha-$PATCH'
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+
+categories:
```

Comment:
Why don't you use the standard Release Drafter configuration?

Reply:
Got it from https://github.com/release-drafter/release-drafter#example. I personally think it is more informative, and uses labels to automatically put the change in the correct category. 
I am not aware of the standard one, but I don't have any strong preferences :)

Reply:
@oleg-nenashev Would you recommend using this https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.github/release-drafter.yml ?

Reply:
Yes. Such configuration basically uses the organization-wide https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml configuration